### DISCUSSION TITLE
Works great in my Pi5

### DISCUSSION THREAD

**Discussion by kjs-DE** [smartavionics/Cura#252] [General]
4.20.20 works great on my Raspberry Pi5 and so far I don't miss any features from the 5.7.0 I had on my PC. The only drawback ist the update checker as it checks with the Ultimaker site and not here on github.
Great job and thank you for providing it!

**Comment by smartavionics** [comment]
Hi, thanks for the feedback. I will tweak the update checker so that it reports my updates.

**Reply by kjs-DE** [reply]
Great and many thanks! To be honest, I can't figure out where 5.7.0 from Ultimaker is better than the version you have.

**Reply by kjs-DE** [reply]
That's what I like with open source! Improvements don't take years but weeks only
